cmd/edit.go

```diff
@@ -35,30 +32,32 @@ func newEditProjectCmd() *cobra.Command {
 
 	editProjectCmd := &cobra.Command{
 		Use:   "edit",
-		Short: "This command will edit the configuration of the PROJECT file",
-		Long:  `This command will edit the configuration of the PROJECT file`,
+		Short: "This command will edit the configuration of the configuration file",
+		Long:  `This command will edit the configuration of the configuration file`,
 		Example: `
 		# To enable the multigroup layout/support
 		kubebuilder edit --multigroup
 		
 		# To disable the multigroup layout/support
 		kubebuilder edit --multigroup=false`,
 		Run: func(cmd *cobra.Command, args []string) {
-			dieIfNoProject()
+			util.DieIfNotConfigured()
 
-			projectInfo, err := scaffold.LoadProjectFile("PROJECT")
+			projectConfig, err := config.LoadFrom(config.DefaultPath)
```

Comment:
```suggestion
			projectConfig, err := config.LoadFromDefaultPath()
```

Reply:
I tried to avoid calling it "project" file, as "PROJECT" is just the file name. It is a configuration file, "project" is not descriptive. Imagiune that in the future, the file name needs to be changed as it conflicts with something, the code won't have tons of variable and functions with "project" just because in previous versions the file was called like that.

Not using "file" was also a design decission. Lets say that, in the future, we no longer use a file for storing config, but we move to another storage solution, which right now sounds very unlikely, but in the future may happen.

So basically my choice was to abstract from both the name and the type of storage, so that in the future they can be changed without the need to update the whole repository. Does it make sense?

But I do like the suggestion of having a function that loads from the default path instead of having to call `config.LoadFrom(config.DefaultPath)`. Wil definetely apply this.